contracts/interfaces/IAxelarAmplifierGateway.sol

```diff
@@ -9,6 +9,32 @@ import { IAxelarGMPGateway } from './IAxelarGMPGateway.sol';
  * @dev Interface for the Axelar Gateway that supports general message passing and contract call execution.
  */
 interface IAxelarAmplifierGateway is IAxelarGMPGateway {
+    /*********\
+    |* Types *|
+    \*********/
+
+    enum CommandType {
+        ApproveContractCall,
+        TransferOperatorship
+    }
+
+    struct Command {
+        CommandType commandType;
+        string messageId;
```

Comment:
Let's make it `bytes` or `bytes32` instead of string. To keep it more efficient as we did with ITS.

And what is it used for?

Reply:
If the length <= 31  we can just store it padded as `bytes32`

Reply:
Oh I see
```
bytes32 commandId = keccak256(bytes(command.messageId));
```

Reply:
We basically repurposed old `payloadHash, sourceTxHash, sourceEventIndex` into the `messageId`. Yeah lets just use `bytes` instead. String encoding (UTF) might mess the binary data

Reply:
`messageId` is an opaque string in the core protocol. For e.g. it's hex encoded tx hash with event index as string for EVM but can be different for other chains. We want to preserve that. Makes it easier to debug on the explorer as well.